source/declarations.tex

```diff
@@ -8876,7 +8876,7 @@
 \pnum
 \recommended
 The use of a fallthrough statement should suppress
-a warning that an implementation might otherwise issue
+a warning that an implementation would otherwise issue
```

Comment:
```suggestion
any warning that an implementation would otherwise issue
```
(To avoid the suggestion that there would necessarily be such a warning.)